app/src/main/res/layout/view_horizontal_scroll_list_item_card.xml

```diff
@@ -14,6 +14,7 @@
             android:id="@+id/horizontal_scroll_list_item_image"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
+            android:scaleType="centerCrop"
```

Comment:
Previously, the FaceDetect and/or WhiteBackground transformations were also applying CENTER_CROP, which was not obvious. Let's allow individual ImageViews to specify their own `scaleType`, which Coil respects automatically.